TheElements/ios/src/main/java/org/moe/samples/elements/ios/AtomicElementViewController.java

```diff
@@ -162,7 +162,6 @@ private CGRect getRectForSubViews(CGSize superviewSize) {
         return viewRect;
     }
 
-    @Override
```

Comment:
Why did you removed the annotation? Since it is a callback method, it should override a super method.